resources/profiles/Zonestar.ini

```diff
@@ -192,33 +213,264 @@ top_infill_extrusion_width = 0.4
+layer_height = 0.40
+bottom_solid_layers = 3
+top_solid_layers = 3
+bridge_flow_ratio = 0.95
+support_material_contact_distance = 0.4
+
+[print:*0.5mm_Layer*]
+layer_height = 0.5
+bottom_solid_layers = 2
+top_solid_layers = 3
+support_material_contact_distance = 0.5
+
+[print:*0.6mm_Layer*]
+layer_height = 0.6
+bottom_solid_layers = 2
+top_solid_layers = 2
+support_material_contact_distance = 0.6
+
+#====================================
+# print preset of nozzle size
+#====================================
+[print:*0.2mm_nozzle*]
+external_perimeter_extrusion_width = 0.22
+extrusion_width = 0.22
+first_layer_height = 0.15
+first_layer_speed = 15
+first_layer_extrusion_width =0.25
+infill_extrusion_width = 0.22
+perimeter_extrusion_width = 0.22
+solid_infill_extrusion_width = 0.22
+top_infill_extrusion_width = 0.22
+gcode_resolution = 0.006
+
+[print:*0.3mm_nozzle*]
+external_perimeter_extrusion_width = 0.33
+extrusion_width = 0.33
+first_layer_height = 0.2
+first_layer_speed = 15
+first_layer_extrusion_width =0.35
+infill_extrusion_width = 0.33
+perimeter_extrusion_width = 0.33
+solid_infill_extrusion_width = 0.33
+top_infill_extrusion_width = 0.33
+gcode_resolution = 0.006
+
+[print:*0.4mm_nozzle*]
+external_perimeter_extrusion_width = 0.45
+extrusion_width = 0.45
+first_layer_height = 0.3
+first_layer_speed = 15
+first_layer_extrusion_width =0.5
+infill_extrusion_width = 0.45
+perimeter_extrusion_width = 0.45
+solid_infill_extrusion_width = 0.45
+top_infill_extrusion_width = 0.45
+gcode_resolution = 0.008
+
+[print:*0.6mm_nozzle*]
+external_perimeter_extrusion_width = 0.6
+extrusion_width = 0.65
+first_layer_height = 0.4
+first_layer_speed = 20
+first_layer_extrusion_width =0.65
+perimeter_extrusion_width = 0.6
+infill_extrusion_width = 0.65
+solid_infill_extrusion_width = 0.65
+top_infill_extrusion_width = 0.65
+gcode_resolution = 0.008
+
+[print:*0.8mm_nozzle*]
+external_perimeter_extrusion_width = 0.8
+extrusion_width = 0.88
+first_layer_height = 0.5
+first_layer_speed = 20
+first_layer_extrusion_width = 0.88
+perimeter_extrusion_width = 0.8
+infill_extrusion_width = 0.88
+solid_infill_extrusion_width = 0.88
+top_infill_extrusion_width = 0.88
+gcode_resolution = 0.008
+
+[print:*1.0mm_nozzle*]
+external_perimeter_extrusion_width = 1.1
+extrusion_width = 1.1
+first_layer_height = 0.6
+first_layer_speed = 25
+first_layer_extrusion_width = 1.1
+perimeter_extrusion_width = 1.0
+infill_extrusion_width = 1.1
+solid_infill_extrusion_width = 1.1
+top_infill_extrusion_width = 1.1
+gcode_resolution = 0.01
+
+[print:*1.2mm_nozzle*]
+external_perimeter_extrusion_width = 1.3
+extrusion_width = 1.3
+first_layer_height = 0.8
+first_layer_speed = 25
+first_layer_extrusion_width = 1.3
+perimeter_extrusion_width = 1.3
+infill_extrusion_width = 1.2
+solid_infill_extrusion_width = 1.3
+top_infill_extrusion_width = 1.3
+gcode_resolution = 0.01
+
+##################
+#print preset    #
+# one color      #
+##################
+[print:0.2 nozzle default]
```

Comment:
All print preset names must contain some specific distinction, for example `@Zonestar`  (it was like that in previous profiles). The main reason is, that preset names must be unique within all PrusaSlicer. Don't forget, you will also have to change names in inheritances.

Example:
`0.2 nozzle default @Zonestar`